### ISSUE TITLE
Script for generating public symbols crashes

### ISSUE THREAD

**Issue by AlmightyYakob** [girder/girder#3069]
When running python scripts/publicNames.py to generate the new public symbols list, the script crashes, outputting this error:

```
Traceback (most recent call last):
  File "scripts/publicNames.py", line 129, in <module>
    main()
  File "scripts/publicNames.py", line 121, in main
    _get_subtree_hierarchy(baseTree, d.split(os.sep)))
  File "scripts/publicNames.py", line 84, in addDirSymbols
    addFileSymbols(subPath, symbolTree[subNameBase])
  File "scripts/publicNames.py", line 54, in addFileSymbols
    symbolScopeKind, symbolScope = symbolExtensions[1].split(':')
ValueError: not enough values to unpack (expected 2, got 1)

```

**Comment by brianhelba** [comment]
Are you using `universal-ctags` or `exuberant-ctags`?

`universal-ctags` is what the script requires, but it's difficult to install in some cases. See #2933

**Comment by AlmightyYakob** [comment]
I am using exuberant-ctags, I'll try to install universal-ctags.

**Comment by zachmullen** [comment]
I am seeing the same error when using universal-ctags on MacOS

**Comment by zachmullen** [comment]
For me, the bug is fixed by 2b2b2fe

We can cherry-pick that onto its own topic, but for now I've put it inside of #2787, which I am hoping can be reviewed and merged.
